qcelemental/models/results.py

```diff
@@ -151,10 +156,7 @@ def _assert1d(cls, v, values):
             raise ValueError("Vector must be castable to shape (-1, )!")
         return v
 
-    @validator(
-        'scf_orbitals_a',
-        'scf_orbitals_b',
-        whole=True)
+    @validator('scf_orbitals_a', 'scf_orbitals_b', whole=True)
     def _assert2d_nao_x(cls, v, values):
```

Comment:
scf_orbitals_b could be empty if restricted = True

Reply:
Sure, this is accounted for since it happens *after* initial validation.